packages/wp-story-editor/src/components/documentPane/index.js

```diff
@@ -44,4 +46,32 @@ function DocumentPane() {
   );
 }
 
+export function PublishModalDocumentPane() {
+  return (
+    <>
+      <PublishPanel name={`story_details_${WP_DOCUMENT_PANEL_NAMES.PUBLISH}`} />
+      <SlugPanel name={`story_details_${DOCUMENT_PANEL_NAMES.SLUG}`} />
+      <PageAdvancementPanel
+        name={`story_details_${DOCUMENT_PANEL_NAMES.PAGE_ADVANCEMENT}`}
+      />
+      <BackgroundAudioPanel
+        name={`story_details_${DOCUMENT_PANEL_NAMES.BACKGROUND_AUDIO}`}
+      />
+      <TaxonomiesPanel
+        name={`story_details_${DOCUMENT_PANEL_NAMES.TAXONOMIES}`}
+      />
```

Comment:
🤔  

I get that overriding the name isn't super intuitive, however it's the one thing that controls the panel collapse state for local storage so it will need an override. This is also the only place that the panels are getting reused, so introducing a new piece of context for it seems more complicated, my opinion. Plus, we'll have to communicate here through each of these that they are part of the modal since it's the only access point that shows the separation. 

The `Panel` component is already doing so much. I get not wanting to spread `...rest`, would you be more ok with this if it's an optional name override in the individual panels, such as `name={overrideName || defaultName}` and a note's included?

Reply:
> so introducing a new piece of context for it seems more complicated

To clarify, I am not suggesting introducing a new piece of (React) context. Just using a separate prop for prefixing the `name` instead of completely overriding it.

Re-read my comment and replace "context" with "prefix" if that makes it more clear.

Reply:
gotcha, i think we're on the same page then. https://github.com/GoogleForCreators/web-stories-wp/pull/10518/commits/1e7113358135c217423dd8e32f49abe24d76f92a